phpcs.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0"?>
-<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="mot-cpms-client" xsi:noNamespaceSchemaLocation="./vendor/squizlabs/php_codesniffer/phpcs.xsd">
```

Comment:
I think this will need flipping back once the scripts etc. are re-added

Reply:
Yeah we will add scripts in a new PR